### PR TITLE
fix(tests): Tests time out and fail

### PR DESCRIPTION
**Pull request by tuckerzp**

### CODE REVIEW

File: packages/oscal-react-library/src/components/OSCALProfile.test.js [hunk 1]
```diff
@@ -1,41 +1,38 @@
...
 
-function testOSCALProfile(parentElementName, renderer) {
-  test(`${parentElementName} displays controls`, async () => {
-    renderer();
     const result = await screen.findByText("AC-1", {
       timeout: 30_000,
     });
     expect(result).toBeVisible();
-  });
+  }, 30_000);
```
Comment (tuckerzp) [inline packages/oscal-react-library/src/components/OSCALProfile.test.js:21]: If we have tests failing, I think this increases the timeout for a given test.

I wonder if the test timeout needed to be >= the findByText timeout.

**Comment by tuckerzp** [comment]
Also if this passes, hold of on merging. I would like to run the tests a few time to be more sure things are working.

File: packages/oscal-react-library/src/components/OSCALProfile.js [hunk 2]
```diff
@@ -39,20 +39,24 @@ export default function OSCALProfile(props) {
...
+          console.log("----- FAILURE -----");
           setError(error);
           setIsLoaded(true);
           props.onResolutionComplete();
         }
       }
     );
 
+    console.log("----- PROFILE RESOLVED ------");
+
```
Comment (tuckerzp) [inline packages/oscal-react-library/src/components/OSCALProfile.js:59]:
```suggestion
```

File: packages/oscal-react-library/src/components/OSCALProfile.js [hunk 3]
```diff
@@ -39,20 +39,24 @@ export default function OSCALProfile(props) {
...
         if (!unmounted.current) {
+          console.log("----- SUCCESS -----");
           setIsLoaded(true);
           setInheritedProfilesAndCatalogs(profilesCatalogsTree);
           props.onResolutionComplete();
         }
       },
       () => {
         if (!unmounted.current) {
+          console.log("----- FAILURE -----");
```
Comment (tuckerzp) [inline packages/oscal-react-library/src/components/OSCALProfile.js:50]:
```suggestion
```

File: packages/oscal-react-library/src/components/OSCALProfile.js [hunk 4]
```diff
@@ -39,20 +39,24 @@ export default function OSCALProfile(props) {
       props.parentUrl,
       (profilesCatalogsTree) => {
         if (!unmounted.current) {
+          console.log("----- SUCCESS -----");
```
Comment (tuckerzp) [inline packages/oscal-react-library/src/components/OSCALProfile.js:42]:
```suggestion
```

**Review by tuckerzp — COMMENTED** [review]
Remove these before merging

**Comment by tuckerzp** [comment]
The tests have now passed 5 times in a row. I am not certain this fixes them completely (we may just need to redo profile resolution for the tests to always pass), but this at least cleans the tests up a little bit.

**Review by kylelaker — APPROVED** [review]

**Pull request merged**